orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/pipeline/BakeStage.groovy

```diff
@@ -94,7 +94,8 @@ class BakeStage implements StageDefinitionBuilder {
     Set<String> deployRegions = (stage.context.region ? [stage.context.region] : []) as Set<String>
     deployRegions.addAll(stage.context.regions as Set<String> ?: [])
 
-    if (!deployRegions.contains("global")) {
+    Boolean autoDetectRegions = Boolean.FALSE != stage.context.autoDetectRegions
```

Comment:
nit: I think most people's expectation is that values not present will be treated as `false`. So I would prefer this to be 'skipRegionDetection` (or something like that) instead (and then you'd test for `true` instead)